api/android/api/src/main/jni/nnstreamer-native-api.c

```diff
@@ -506,6 +505,23 @@ Java_org_nnsuite_nnstreamer_NNStreamer_nativeInitialize (JNIEnv * env, jclass cl
   return JNI_TRUE;
 }
 
+/**
+ * @brief Native method to initialize NNStreamer.
+ */
+jboolean
+Java_org_nnsuite_nnstreamer_NNStreamer_nativeInitialize (JNIEnv * env, jclass clazz,
+    jobject context)
+{
+  nns_logi ("Called native initialize.");
+
+  if (!gst_is_initialized ()) {
+    nns_loge ("GStreamer is not initialized.");
+    return JNI_FALSE;
+  }
+
+  return nnstreamer_native_initialize ();
```

Comment:
calling ```gst_init_check``` after calling ```gst_is_initialized```?

Reply:
I will move gst_is_initialized() into nnstreamer_native_initialize(), and remove ml_initialize_gstreamer().
(GStreamer.java should be called and ml_initialize_gstreamer() is unnecessary.)

Reply:
Updated